### PR TITLE
Added swift versioning test case

### PR DESCRIPTION
**Pull request by udaysk23**
Added swift versioning test case

Signed-off-by: udaysk23 <ukurundw@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: suites/pacific/rgw/sanity_rgw.yaml [hunk 1]
```diff
@@ -180,6 +180,16 @@ tests:
         config-file-name: test_swift_basic_ops.yaml
         timeout: 300
 
+  - test:
+      name: swift versioning copy tests
+      desc: restore versioned object in swift
+      polarion-id: CEPH-10646
```
Comment (rakeshgm) [inline suites/pacific/rgw/sanity_rgw.yaml:186]: This script points to for s3 here, [CEPH-10646](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-10646) and not swift versioning test, can you take a look at it and update. but do not remove s3 test script in polarion.

**Comment by rakeshgm** [comment]
add the appropriate labels and attach the logs please.

**Comment by rakeshgm** [comment]
did you run the tests, can you attach the logs here. ?.

**Comment by udaysk23** [comment]
> did you run the tests, can you attach the logs here. ?. use our jenkinks portal to trigger the runs. its quite easy

I have triggered suite in Jenkins, will update log once it get completed

File: suites/pacific/rgw/tier_1_object.yaml [hunk 2]
```diff
@@ -97,3 +97,12 @@ tests:
...
         timeout: 500
+  - test:
+      name: swift versioning copy tests
+      desc: restore versioned object in swift
+      polarion-id: CEPH-10646
+      module: sanity_rgw.py
+      config:
+        script-name: test_swift_basic_ops.py
+        config-file-name: test_swift_version_copy_op.yaml
+        timeout: 500
```
Comment (sunilkumarn417) [inline suites/pacific/rgw/tier_1_object.yaml:108]: please add newline at EOF.

**Comment by udaysk23** [comment]
Script is failing on 5.x cluster
https://bugzilla.redhat.com/show_bug.cgi?id=1966100

**Comment by rakeshgm** [comment]
nice one @udaysk23

**Comment by udaysk23** [comment]
Will rerun suite once following PR get merged into ceph-qe-script

https://github.com/red-hat-storage/ceph-qe-scripts/pull/130

**Comment by psathyan** [comment]
@udaysk23 can we work on closing this PR?

**Comment by udaysk23** [comment]
Following is the Jenkis link of successful run
https://ceph-downstream-jenkins-csb-storage.apps.ocp4.prod.psi.redhat.com/job/CEPH-RGW/36/console

File: suites/pacific/rgw/sanity_rgw.yaml [hunk 3]
```diff
@@ -180,6 +180,16 @@ tests:
...
+      module: sanity_rgw.py
+      config:
+        script-name: test_swift_basic_ops.py
+        config-file-name: test_swift_version_copy_op.yaml
+        timeout: 500
+
```
Comment (psathyan) [inline suites/pacific/rgw/sanity_rgw.yaml:192]: IMO, we should not add the suite to sanity if it is already added to `tier-1`. @rakeshgm your thoughts?

Comment (udaysk23) [reply suites/pacific/rgw/sanity_rgw.yaml]: I have removed code from sanity suite

**Review by psathyan — APPROVED** [review]

**Pull request merged**